README.md

```diff
@@ -1,2 +1,25 @@
-# global-streetscapes
-Retrieving and using data from the global-streetscapes dataset. 
+# streetscapes
+This repository contains information and code for retrieving and using data from the [global-streetscapes](https://github.com/ualsg/global-streetscapes/tree/main) dataset. 
+
+### Converting global-streetscapes to parquet
+
+`convert_csv_to_parquet.py` contains the code for reproducing `streetscapes-data.parquet` dataset. 
+For input you will have to download the original csv files from [hugging face](https://huggingface.co/datasets/NUS-UAL/global-streetscapes/tree/main/data).
+To limit size, the dataset contains the following data:
+    - `contextual.csv`
+    - `metadata_common_attributes.csv`
+    - `segmentation.csv`
+    - `simplemaps.csv`
+
```

Comment:
It might be obvious, but perhaps it would be good to say that these should go into the `streetscapes-data` directory.